hooks/tk-nuke_actions.py

```diff
@@ -183,15 +183,21 @@ def _create_read_node(self, path, sg_publish_data):
         if ext.lower() not in valid_extensions:
             raise Exception("Unsupported file extension for '%s'!" % path)
 
+        # This form will extract the format and frame range from the file
+        # itself. We can also see if there's a matching template and override
+        # the frame range, but this should handle the zero config case. This
+        # will also automatically extract the format and frame range for movie
+        # files.
+        read_node = nuke.createNode("Read")
```

Comment:
If there's a technical reason for doing it this way instead of the other, which to an untrained eye looks the same, maybe its worth pointing it out in the code. Otherwise you run the risk of someone seeing that code and think "Hey, why is this not simply using `nuke.nodes.ReadNode`, it would be a single line of code instead of three."